src/jdk.httpserver/share/classes/com/sun/net/httpserver/HttpExchange.java

```diff
@@ -66,6 +65,9 @@
 
 public abstract class HttpExchange implements AutoCloseable {
 
+    /**
+     * Creates a HttpExchange.
```

Comment:
I'd suggest to keep consistent... You have been using `* Constructor for subclasses to call` at other places. Maybe you should use that sentence here too.

Reply:
Replaced wording -- now using `* Constructor for subclasses to call`